src/js/controller/controller.js

```diff
@@ -1229,6 +1219,34 @@ Object.assign(Controller.prototype, {
         }, model);
 
         this.trigger(ERROR, evt);
+    },
+    updatePlaylist(playlist, feedData) {
+        if (this.extendedUpdatePlaylist) {
```

Comment:
Where is this defined?

Reply:
https://github.com/jwplayer/jwplayer-commercial/pull/7934/files#diff-15fcddfc5fb5f2333b37e0588e65df209035661bb83f68a5e4d0637c2b260e92R81

Reply:
@pajong that gave me an idea on how to clean it up, this is now handled in commercial by renaming methods on the prototype.

https://github.com/jwplayer/jwplayer-commercial/pull/7934/files#diff-15fcddfc5fb5f2333b37e0588e65df209035661bb83f68a5e4d0637c2b260e92R17-R103

I removed `extendedUpdatePlaylist`  and `extendedSetup` from these changes.